aten/src/ATen/native/Resize.cpp

```diff
@@ -35,7 +35,8 @@ static bool is_functorch_wrapped_tensor(const Tensor& tensor) {
   return !(key_set & kFunctorchWrappedTensors).empty();
 }
 
-bool resize_output(const Tensor& output, IntArrayRef shape) {
+bool resize_output(const Tensor& output, IntArrayRef shape,
+    c10::optional<MemoryFormat> optional_memory_format /* = c10::nullopt */) {
```

Comment:
I don't think we want this behavior, `optional_memory_format` will be applied only if the `output` shape doesn't match with the match shape. (cc: @ngimel for opinion)

Similar discussion: https://github.com/pytorch/pytorch/issues/49878#issuecomment-752191914

Reply:
@ngimel and I had an opportunity to discuss this offline, here's our thinking:

- memory format is like strides, if a tensor has to be resized then both it and strides can be reset
- if a tensor is the correct shape, however, then strides and memory format should be preserved 

So I don't think we need to pass a memory format argument to resize_output

Reply:
Resolved and discussed [here](https://github.com/pytorch/pytorch/pull/62560#discussion_r687363362).